lib/config/option.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2009  The "config" Authors
```

Comment:
move `lib` to `third_party` if code is from 3rd party library

Reply:
follow your suggestion